examples/benchmark_mipnerf360_stmt.py

```diff
@@ -0,0 +1,115 @@
+# Training script for the Mip-NeRF 360 dataset
+# The model is trained with downsampling factor 8 and rendered with downsampling factor 1, 2, 4, 8
+
+import os
+import GPUtil
+from concurrent.futures import ThreadPoolExecutor
+import time
+import glob
+
+# 9 scenes
+# scenes = ["bicycle", "bonsai", "counter", "flowers", "garden", "stump", "treehill", "kitchen", "room"]
+# factors = [4, 2, 2, 4, 4, 4, 4, 2, 2]
+
+# 7 scenes
+scenes = ["bicycle", "bonsai", "counter", "garden", "stump", "kitchen", "room"]
+factors = [8] * len(scenes)
+
+excluded_gpus = set([])
+
+# classic
+result_dir = "results/benchmark_stmt"
```

Comment:
what does `stmt` stands for? lol

Reply:
I think` Single-scale-Training-and-Multi-scale-Testing`, the idea being you train with a fixed resolution (here a factor of 1/8 of the original image res) and test on various other resolutions (1x full res,  1/2, 1/4, 1/8).